intranet/apps/auth/backends.py

```diff
@@ -141,16 +140,9 @@ def authenticate(self, request, username=None, password=None):
         # remove all non-alphanumerics
         username = re.sub(r"\W", "", username).lower()
 
-        krb_ticket, ad_auth = self.get_kerberos_ticket(username, password)
-
-        if krb_ticket == "reset":
-            user, _ = get_user_model().objects.get_or_create(username="RESET_PASSWORD", user_type="service", id=999999)
-            return user
+        result, ad_auth = self.get_kerberos_ticket(username, password)
 
-        if not krb_ticket:
-            kerberos_authenticate_failures.inc()
-            return None
-        else:
+        if result == KerberosAuthenticationResult.SUCCESS:
             logger.debug("Authentication successful")
             try:
```

Comment:
This is just my personal preference, but I like having a blank line before and after a `try-except` block.